session_db/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+Store sessions in a database instead of the filesystem.
```

Comment:
If possible, could you mention the interest to store sessions in db instead of filesytem ?

Reply:
@legalsylvain I added this to the README: *This simplifies the configuration of horizontally scalable deployments, by avoiding the need for a
distributed filesystem to store the Odoo sessions.*